libs/langgraph/src/graph/state.ts

```diff
@@ -180,6 +187,28 @@ export class StateGraph<
     ]);
   }
 
+  _addSchema(stateDefinition: StateDefinition) {
+    for (const [key, val] of Object.entries(stateDefinition)) {
+      let channel;
+      if (typeof val === "function") {
+        channel = val();
+      } else {
+        channel = val;
+      }
+      if (this.channels[key] !== undefined) {
+        if (this.channels[key].lc_graph_name !== channel.lc_graph_name) {
+          if (channel.lc_graph_name !== "LastValue") {
```

Comment:
I don't think we have this error like this in py?

Reply:
Fixed